docs/content/tutorial/step_15.ngdoc

```diff
@@ -0,0 +1,327 @@
+    "karma-jasmine": "^0.3.8",
+    "protractor": "^3.2.2",
+    "protractor-accessibility-plugin": "^0.1.1",
+    "shelljs": "^0.6.0"
+   }
+}
+```
+
+Now, we must tell bower to download and install these dependencies.
+
+```
+npm install
+```
+
+<div class="alert alert-info">
+  **Note:** If you have bower installed globally, you can run `bower install`, but for this project
+  we have preconfigured `npm install` to run bower for us.
+</div>
+
+<div class="alert alert-warning">
+  **Warning:** If a new version of Angular has been released since you last ran `npm install`, then
+  you may have a problem with the `bower install` due to a conflict between the versions of
+  angular.js that need to be installed. If you run into this issue, simply delete your
+  `app/bower_components` directory and then run `npm install`.
+</div>
+
+
+## How to use the `ngAria` module
+
+The `ngAria` module provides support for common ARIA attributes that convey state or semantic information about the application for users of assistive technologies, such as screen readers.
+
+### Template
+
+In order to enable the module, we need to update `index.html`, loading the necessary dependencies
+(**angular-aria.js**) that contain JavaScript code. 
+
+The WAI-ARIA module, {@link ngAria ngAria}, contains the code necessary to help you to add WAI-ARIA attributes in your application.
+
+<br />
+**`app/index.html`:**
+
+```html
+  ...
+  <!-- Adds WAI-ARIA module in AngularJS -->
+  <script src="bower_components/angular-aria/angular-aria.js"></script>
+  ...
+```
+
+### Dependencies
+
+We need to add a dependency on `ngAria` to our main module first:
+
+<br />
+**`app/app.module.js`:**
+
+```js
+  angular.
+    module('phonecatApp', [
+      ...
+      'ngAria',
+      ...
+    ]);
+```
+
+## How to inform the user that _something_ is happening
+
+Today, is very common to find applications or webpages where the user make some interaction with an element and obtains and inmediate reply to the action.
+
+The problem is that in most of the cases this action is only reported in a way that you need to see it in your screen but if a user is navigating with a screenreader like [JAWS](http://www.freedomscientific.com/Products/Blindness/JAWS), [NVDA](http://www.nvaccess.org/) or [ChromeVox](http://www.chromevox.com/), this information never won't be accessible.
+
+When the user is searching or filtering mobiles in the application, the list updates automatically. It is mandatory that user knows, in some way, the result after the search or order action.
+
+To fix this, we add two directives to inform the user how many items are know in the list after search or order them. This directive will fill an element in our page to tell the user how many elements are after the action.
+
+<br />
+**`app/phone-list/phone-list-template.html`:**
+
+```html
+  <div>
+    <label for="search">Search:</label> 
+    <input id="search" ng-model="$ctrl.query" ng-model-options="{ debounce: 900 }">
```

Comment:
My first attempt was not to use the debounce, but without it, there problems to inform the user about the number of results. The information that the user obtains is not correct. It is neccesary an small "gap" to get the results, and update the aria region.

I can try to reduce this number to the mininum required.